### PR TITLE
[21890] Update commercial support section in project README

### PR DESCRIPTION
**Pull request by JesusPoderoso**
This PR updates this project README's commercial support section

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Linux/622/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Windows/642/)

File: README.md
Comment (LuciaEchevarria99) [inline README.md:1]: The `Getting Help` section needs to be deleted

**Review by LuciaEchevarria99 — CHANGES_REQUESTED** [review]

**Review by LuciaEchevarria99 — APPROVED** [review]
LGTM

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Linux/623/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Windows/643/)

**Pull request merged**